stdlib/public/core/Filter.swift.gyb

```diff
@@ -268,6 +268,22 @@ public struct ${Self}<
     return ${Slice}(base: self, bounds: bounds)
   }
 
+  // Any estimate of the number of elements that pass `_predicate` requires
+  // iterating the collection and evaluating each element, which can be costly,
+  // is unexpected, and usually doesn't pay for itself in saving time through
+  // preventing intermediate reallocations. (SR-4164)
+  public var underestimatedCount: Int { return 0 }
+
+  public func _copyToContiguousArray()
```

Comment:
Rather than doing this, wouldn't it be better to have _copyCollectionToContiguousArray use underestimatedCount?